data_steward/analytics/cdr_ops/controlled_tier_qc/check_controlled_tier.py

```diff
@@ -20,15 +20,19 @@
 post_deid_dataset: str = ""  # the CT deid dataset
 pre_deid_dataset: str = ""  # the combined dataset
 mapping_dataset: str = ""  # the sandbox dataset where mappings are
-# -
 
+# +
 import pandas as pd
 from code.controlled_tier_qc import run_qc, display_check_summary_by_rule, display_check_detail_of_rule
 
+pd.set_option('display.max_colwidth', -1)
+pd.set_option('display.width', None)
+# -
+
 # to_include = ['DC-1370', 'DC-1377', 'DC-1346', 'DC-1348', 'DC-1355', 'DC-1357', 'DC-1359', 
 #             'DC-1362', 'DC-1364', 'DC-1366', 'DC-1368', 'DC-1373',
 #             'DC-1380', 'DC-1382', 'DC-1388', 'DC-1496']
-to_include = []
+to_include = ['DC-1527', 'DC-1535', 'DC-1370', 'DC-1377', 'DC-1362']
```

Comment:
Please set `to_include = []` so that all rules will run by default. Engs typically run this from the command line and will not think to edit this line.